vscode/src/extension.ts

```diff
@@ -181,8 +182,24 @@ async function activateLanguageService(extensionUri: vscode.Uri) {
     ...registerQSharpNotebookCellUpdateHandlers(languageService),
   );
 
+  // format document
+  const isFormattingEnabled = getEnableFormating();
+  const formatterEvent = {
+    event: undefined as vscode.Disposable | undefined,
```

Comment:
Calling this 'formatterEvent' (and the member 'event') confused me for a bit. There's not any event handling here, right? Or even any event-related objects. It's just a container to hold the Disposable returned from registering the edit provider (if I'm reading the code correctly). I clearer name would be less confusing.

Reply:
renamed to FormatterHandle